specs/avatar.md

```diff
@@ -0,0 +1,105 @@
+# Avatar
+
+## Overview
+
+An avatar is a graphical represention of a user or object.
+
+### Use Cases
+
+A common use case would be to display an image or text (usually initials) of a user or an object.
+
+### Features
+- Image slot: Able to slot in an avatar image
+- Badge slot: Able to slot in a badge component
+- `size`, a numeric value that determines the size of the avatar
```

Comment:
I think this is the first time we have an instance of `size` in one of our components. Is this something we want to do broader thinking about? For example, some systems have `size` for buttons as well. If we want to enable that, I think we'd want to do it in a consistent way.

Reply:
Also, is there a minimum size? For example, I noticed in some examples that when the size is small enough the image/coin is hidden entirely and only the badge is displayed.

Reply:
Thanks for the feedback. This is great. I'll think through this a little bit more and submit something.

Reply:
I've tabled the addition of a `size` attribute until a broader sizing convention decision can be reached.